src/components/Form/FetchDocumentForm.jsx

```diff
@@ -63,7 +63,7 @@ const FetchDocumentForm = () => {
       <form onSubmit={handleGetDocumentSubmission} autoComplete="off">
         <div style={formRowStyle}>
           <label htmlFor="search-doctype">Select document type to search: </label>
-          <DocumentSelection htmlId="search-doctype" />{' '}
+          <DocumentSelection htmlId="search-doctype" />
```

Comment:
We might leave this to the UI team since it's more related to styling. But without the spacing, the `<select>` element and the `button` element is squashed together. (see below)

<img width="515" alt="Screen Shot 2023-05-11 at 0 58 59" src="https://github.com/codeforpdx/PASS/assets/14917816/8652f086-4b44-4ca1-a170-b8926d733efb">

Perhaps we can leave a TODO comment or something related related to DocumentSelection and button in the forms or temporarily leave an `&nbsp;` if not the {' '}

Reply:
I removed these updates, altough really UX should be using CSS for this kind of spacing.

Reply:
> I removed these updates, altough really UX should be using CSS for this kind of spacing.

I agree. I was a bit confused when I saw manual spacing being added like this, although I know that some Prettier configs will result in these kinds of things being entered into the code when you run the Prettier check. Not sure if that's what happened here or if it was manually entered.

Reply:
So one of the things I accidentally broke with Ka Hung's eslint file was the prettier configuration. I fixed it now, and these weird curly spaces are gone.

Reply:
Hmm seems like if there's leftover space at the end of a line in jsx, prettier will escape it like this, since it has syntactic significance in JSX. We probably don't actually want prettier doing this, and instead probably want the jsx whitespace removed. Not sure of the best way to fix it though, since the code is already in development.